src/copygb2/copygb2.F90

```diff
@@ -1349,6 +1349,7 @@ END SUBROUTINE INTGRIB2
   SUBROUTINE INTGRIB1(K1F,KGDS1F,K2F,KGDS2F,MRL,MRO, &
        IV,IP,IPOPT,KGDS1,K1,IB1,L1,F1,G1,KGDS2,K2, &
        IB2,L2,F2,G2,IRET)
+    USE IP_MOD
```

Comment:
OK, last night I realized that these "USE IP_MOD" statements should be wrapped in an #ifdef which checks whether ip-4.0.0 or greater was found.

In this way, the code would continue to work with older versions if ip.